src/Estimators/tests/ValidSpaceGridInput.h

```diff
@@ -30,6 +30,7 @@ struct ValidSpaceGridInput
     SPHERICAL,
     WITH_STEP,
     WITHOUT_STEP,
+    WEIRD_CARTESIAN,
```

Comment:
WEIRD is valid or not? I doesn't sound absolutely wrong. Confused.

Reply:
Well its not super strange but the scaling is different for one dimension. This is perhaps unexciting enough to just drop.  Initially I was trying to have it cover stranger cases but they all turned out to have issues with respect to mapping into the grid properly.  I guess a stranger case would be if the origin was shifted to properly drop the grid onto something other than +/- r_x / scaling.  Currently you can't for instance do 0 1 as it won't cover the minimum image coordinate range.  For all these bad choices of you now get an exception rather than a crash when a memory operation is enough out a bounds.

Reply:
I see

Reply:
CI is busy on another PR. If you have a second, could add a comment
```
    WEIRD_CARTESIAN, # briefly explain the special aspect of this test case and also comment if this input case is valid or not.
```